pinot-core/src/main/java/org/apache/pinot/core/segment/index/loader/defaultcolumn/V3DefaultColumnHandler.java

```diff
@@ -67,10 +67,19 @@ protected void updateDefaultColumn(String column, DefaultColumnAction action, In
     Set<String> textIndexColumns = indexLoadingConfig.getTextIndexColumns();
```

Comment:
Since text column supports both raw & dictionary-encoded index, remove the branch for text column

Reply:
done